packages/accounts/src/light-account/accounts/multiOwner.ts

```diff
@@ -0,0 +1,202 @@
+};
+
+export type CreateMultiOwnerLightAccountParams<
+  TTransport extends Transport = Transport,
+  TSigner extends SmartAccountSigner = SmartAccountSigner,
+  TLightAccountVersion extends LightAccountVersion<"MultiOwnerLightAccount"> = LightAccountVersion<"MultiOwnerLightAccount">,
+  TEntryPointVersion extends GetEntryPointForLightAccountVersion<
+    "MultiOwnerLightAccount",
+    TLightAccountVersion
+  > = GetEntryPointForLightAccountVersion<
+    "MultiOwnerLightAccount",
+    TLightAccountVersion
+  >
+> = Omit<
+  CreateLightAccountBaseParams<
+    TTransport,
+    TSigner,
+    "MultiOwnerLightAccount",
+    TLightAccountVersion,
+    TEntryPointVersion
+  >,
+  "getAccountInitCode" | "entryPoint" | "version" | "abi" | "accountAddress"
+> & {
+  salt?: bigint;
+  initCode?: Hex;
+  accountAddress?: Address;
+  factoryAddress?: Address;
+  version?: TLightAccountVersion;
+  entryPoint?: EntryPointDef<TEntryPointVersion, Chain>;
+  owners?: Address[];
+};
+
+export async function createMultiOwnerLightAccount<
+  TTransport extends Transport = Transport,
+  TSigner extends SmartAccountSigner = SmartAccountSigner,
+  TLightAccountVersion extends LightAccountVersion<"MultiOwnerLightAccount"> = LightAccountVersion<"MultiOwnerLightAccount">,
+  TEntryPointVersion extends GetEntryPointForLightAccountVersion<
+    "MultiOwnerLightAccount",
+    TLightAccountVersion
+  > = GetEntryPointForLightAccountVersion<
+    "MultiOwnerLightAccount",
+    TLightAccountVersion
+  >
+>(
+  config: CreateMultiOwnerLightAccountParams<
+    TTransport,
+    TSigner,
+    TLightAccountVersion,
+    TEntryPointVersion
+  >
+): Promise<
+  MultiOwnerLightAccount<TSigner, TLightAccountVersion, TEntryPointVersion>
+>;
+
+export async function createMultiOwnerLightAccount({
+  transport,
+  chain,
+  signer,
+  initCode,
+  version = defaultLightAccountVersion("MultiOwnerLightAccount"),
+  entryPoint = getEntryPoint(chain, {
+    version:
+      AccountVersionRegistry["MultiOwnerLightAccount"][version]
+        .entryPointVersion,
+  }),
+  accountAddress,
+  factoryAddress = AccountVersionRegistry["MultiOwnerLightAccount"][version]
+    .address[chain.id].factory,
+  salt: salt_ = 0n,
+  owners = [],
+}: CreateMultiOwnerLightAccountParams): Promise<MultiOwnerLightAccount> {
+  const client = createBundlerClient({
+    transport,
+    chain,
+  });
+
+  const getAccountInitCode = async () => {
+    if (initCode) return initCode;
+
+    // NOTE: the current signer connected will be one of the owners as well
+    const ownerAddress = await signer.getAddress();
+    // owners need to be dedupe + ordered in ascending order and not == to zero address
+    const owners_ = Array.from(new Set([...owners, ownerAddress]))
+      .filter((x) => hexToBigInt(x) !== 0n)
+      .sort((a, b) => {
+        const bigintA = hexToBigInt(a);
+        const bigintB = hexToBigInt(b);
+
+        return bigintA < bigintB ? -1 : bigintA > bigintB ? 1 : 0;
+      });
```

Comment:
we duplicate this logic in a lot of places (multisig, multiownerMA, and here). We should create a util for this